doc/user_documentation.rst

```diff
@@ -273,26 +273,37 @@ removed after 40 days of build inactivity.
 Pagure Integration
 ------------------
 
-For any pagure instance (including `src.fedoraproject.org <http://src.fedoraproject.org/>`_), you can setup Copr auto-rebuilding and pr/commit flagging on new changes landing into a pagure repository and its open pull requests.
+For any Pagure instance (including `src.fedoraproject.org <http://src.fedoraproject.org/>`_), you can setup Copr
+auto-rebuilding, triggering build on a PR comment containing ``[copr-build]`` keyword and pr/commit flagging on new
+changes landing into a pagure repository and its open pull requests.
 
 Auto-rebuilding
 ^^^^^^^^^^^^^^^
 
-On the Pagure side, you need to set Fedmsg to 'active' in your project settings (in 'Hooks' section almost at the bottom). For some instances
-(e.g. `src.fedoraproject.org <http://src.fedoraproject.org/>`_), this might already be active by default so you don't even need to perform this step.
+To enable auto-rebuilding on push events, you need to set Fedmsg to 'active' in your project settings on Pagure
+(in 'Hooks' section). For some instances (e.g. `src.fedoraproject.org <http://src.fedoraproject.org/>`_), this
+might already be active by default so you don't even need to perform this step.
```

Comment:
s/already be active by default so you don't even need to perform this step/already be active for all projects./